.github/dockerfile/Dockerfile.debian

```diff
@@ -32,6 +32,9 @@ RUN case $(dpkg --print-architecture) in \
     && wget --no-verbose https://golang.org/dl/go$GO_VERSION.linux-$ARCH.tar.gz \
     && tar -C /usr/local -xzf go$GO_VERSION.linux-$ARCH.tar.gz
 
+RUN wget https://github.com/apple/foundationdb/releases/download/7.3.19/foundationdb-clients_7.3.19-1_amd64.deb \
```

Comment:
When building "full," it will also build "fdb," so the CGO needs to be modified. Changing the "docker.debian" file is necessary because the GitHub workflow includes a check for building "full."

It is also possible to not build "fdb" during the "full" build, so neither of these places would require any modifications.